src/bindings/swig/python/examples/example_keyset.py

```diff
@@ -23,6 +23,29 @@
 ks1.remove("user:/key6")
 print("")
 
+print("Alternatively you can use pop")
+ks1.extend(list_of_keys)
+# pop() removes the last element of the KeySet and returns its value
+print(ks1.pop())
+print(ks1.pop())
+
+print("or cut")
+ks1.append(new_key)
+# cut(kdb::Key) removes a given kdb key from the KeySet and returns its value
+# this method takes the key itself as input parameter in contrast to remove() which takes the path as a string
+print(ks1.cut(new_key))
+print("")
```

Comment:
The comment explaining `cut` is not quite correct. `ks1.cut(new_key)` will also remove any keys **below** `new_key` and returns a new `KeySet` with these keys.

This snippet would show this better:

```suggestion
print("or cut")
ks1.append(kdb.Key("user:/keytocut"))
ks1.append(kdb.Key("user:/keytocut/below"))
# cut(kdb::Key) removes all Keys below a given Key from the KeySet and returns them in a new KeySet
# this method takes a kdb.Key as input parameter in contrast to remove() which takes the path as a string
print(ks1.cut(kdb.Key("user:/keytocut")))
print("")
```

Reply:
added in [9bda518](https://github.com/ElektraInitiative/libelektra/pull/4377/commits/9bda51805c78f7dc9d6f7579b8ff82d52a2b6c6c)